extensions/ql-vscode/test/vscode-tests/cli-integration/databaseFetcher.test.ts

```diff
@@ -0,0 +1,92 @@
+import { join } from "path";
+import { extensions, CancellationToken, Uri, window } from "vscode";
+
+import { CodeQLExtensionInterface } from "../../../src/extension";
+import { CodeQLCliServer } from "../../../src/cli";
+import { DatabaseManager } from "../../../src/databases";
+import {
+  importArchiveDatabase,
+  promptImportInternetDatabase,
+} from "../../../src/databaseFetcher";
+import { cleanDatabases, dbLoc, DB_URL, storagePath } from "./global.helper";
+
+jest.setTimeout(60_000);
+
+/**
+ * Run various integration tests for databases
+ */
+describe("DatabaseFetcher", () => {
```

Comment:
Good idea to move these tests 👍🏽 
Shall we move this commit to a separate PR, so we can get this merged without cluttering up our "actual" change?

Reply:
Can do!